kuiper-game/Systems/SaveLoad.cs

```diff
@@ -11,11 +11,10 @@ namespace Kuiper.Systems
     public static class SaveLoad
     {
         private static string savePath = Directory.CreateDirectory(Path.Join(AppDomain.CurrentDomain.BaseDirectory, "saves")).FullName;
-
+        private static JsonSerializerSettings settings = new JsonSerializerSettings { TypeNameHandling = TypeNameHandling.Auto, PreserveReferencesHandling = PreserveReferencesHandling.Objects};
```

Comment:
Newtonsoft had a hard time with interface types, in this case IShipEvents. Doesn't seem to bloat the save file